code/modules/holiday/easter.dm

```diff
@@ -158,37 +158,26 @@
 	icon_state = "egg-[eggcolor]"
 
 /obj/item/food/egg/proc/dispensePrize(turf/where)
```

Comment:
How do you feel about just making this the arcade toy drop list, plus some additional bunny related items? Just a though, not sure if it's a good idea just for refactoring's sake.

Reply:
i think its kind of out of scope for this, not sure